include/envoy/common/token_bucket.h

```diff
@@ -18,16 +18,19 @@ class TokenBucket {
   virtual ~TokenBucket() {}
 
   /**
-   * @param tokens supplies the number of tokens to be consumed. Default is 1.
-   * @return true if bucket is not empty, otherwise it returns false.
+   * @param tokens supplies the number of tokens to be consumed.
+   * @param allow_partial supplies whether the token bucket will allow consumption of less tokens
+   *                      than asked for. If allow_partial is true, the bucket contains 5 tokens,
+   *                      and the caller asks for 3, the bucket will return 3 tokens.
```

Comment:
Should we clarify here what happens to other two tokens?

Reply:
Oops sorry I typed this wrong, will fix.

Reply:
Thanks. Now it makes sense.